### ISSUE TITLE
Missing form labels- Contact Us

### ISSUE THREAD

**Issue by mariapraetzellis** [CDLUC3/dmptool#366]
On https://dmptool.org/contact-us. 

Applicable WCAG Standard(s)
--
1.1.1 Non-text Content (Level A)
1.3.1 Info and Relationships (Level A)
2.4.6 Headings and Labels (Level AA)
3.3.2 Labels or Instructions (Level A)

**Comment by briri** [comment]
Missing form labels are connected to the hidden (automated) Recaptcha checks. There are 2 hidden text areas on the page that are targets for Recaptcha failures. If a failure occurs then those text areas are displayed. Added labels (only visible to screen readers that explain the purpose of the text areas.

Also removed the redundant alt text from the Spinner.
